project1/models.py

```diff
@@ -0,0 +1,15 @@
+from flask_sqlalchemy import SQLAlchemy
+
+
+db = SQLAlchemy()
+
+
+class User(db.Model):
+      __tablename__ = "users"
+      id = db.Column(db.Integer, primary_key=True)
+      email = db.Column(db.String, nullable=False)
+      password = db.Column(db.String, nullable=False)
+      timestamp = db.Column(db.String,nullable=False)
+
+
+
```

Comment:
Didn't you add ID as serial key? How is this being updated?

Reply:
took id=count=0  and count+ every time I register